<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Serge Poltavski</author>
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>General</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/cuneiform.qrc">
            <normaloff>:/img/oxygen/32x32/select_rectangular.png</normaloff>:/img/oxygen/32x32/select_rectangular.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Editor</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/cuneiform.qrc">
            <normaloff>:/img/oxygen/32x32/object_edit.png</normaloff>:/img/oxygen/32x32/object_edit.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
          <property name="toolTip">
           <string>Debug settings</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/cuneiform.qrc">
            <normaloff>:/img/oxygen/32x32/configure.png</normaloff>:/img/oxygen/32x32/configure.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="generalSettings">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>241</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>General settings</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="editorSettings">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="showCurrentCharacterLabel">
               <property name="text">
                <string>Show current character:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="showCurrentCharacterCheckBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="currentCharacterColorLabel">
               <property name="text">
                <string>Current character color:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ColorButton" name="currentCharacterColorButton"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="alternativesColorLabel">
               <property name="text">
                <string>Alternatives color:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ColorButton" name="alternativeColorButton"/>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="fontChoose">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="fontChooseLabel">
               <property name="text">
                <string>Default font:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="debugSettings">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="printCuneiformDebug">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Print libcuneiform debug information</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="cimage_debug_">
             <property name="text">
              <string>CImage module debugging</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Show formatter bounding boxes for:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="showSectionsBBox">
                <property name="text">
                 <string>sections</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showColumnsBBox">
                <property name="text">
                 <string>columns</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showParagraphsBBox">
                <property name="text">
                 <string>paragraphs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showLinesBBox">
                <property name="text">
                 <string>lines</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showCharactersBBox">
                <property name="text">
                 <string>characters</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showPicturesBBox">
                <property name="text">
                 <string>pictures</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/cuneiform.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
